src/AuthModule/Components/ResetPassword/ResetPassword.tsx

```diff
@@ -1,7 +1,172 @@
-import React from 'react'
+import React, { useState } from 'react'
+import Loader from '../../../SharedMoudule/Components/Loading/Loading'
+import { useNavigate } from 'react-router-dom';
+import logo from "../../../assets/PMS 3.svg";
+import { useForm } from 'react-hook-form';
+import { OTPValidation, emailValidation, passwordValidation } from "../Validator/Validator.js";
+import { baseUrl } from '../../../Constants/Components/Urls';
+import { toast } from 'react-toastify';
+import axios from 'axios';
 
 export default function ResetPassword() {
+
+  const navigate=useNavigate();
+  // ########################## submit New password ##################################
+
+  const [isLoading,setIsLoading]=useState(false);
+  const {register,handleSubmit,formState:{errors}}=useForm();
+
+  const submitNewPassword=async(data:any)=>{
+    try {
+      setIsLoading(true);
+      const response = await axios.post(`${baseUrl}/Users/Reset`,data);
+        console.log(response);
+        toast.success(" reset New password please!")
```

Comment:
We can change this to "password reset successfully", as the current sentence feels like we urging the user to reset the password, but it is already reset.